### PR TITLE
Introduce vehicles, warehouses and trailers

### PR DESCRIPTION
**Pull request by Atala**

### CODE REVIEW

File: src/Entity/Trailer.php [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+/**
+ * @ApiResource(
+ *   attributes={
+ *     "normalization_context"={"groups"={"trailer"}},
+ *     "denormalization_context"={"groups"={"trailer_create"}},
+ *   },
+ *   collectionOperations={
+ *     "get"={
+ *       "method"="GET",
+ *       "access_control"="is_granted('ROLE_DISPATCHER') or is_granted('ROLE_ADMIN')",
```
Comment (vladimir-8) [inline src/Entity/Trailer.php:22]: I think `"access_control"="is_granted('ROLE_DISPATCHER')",` will do the same. As ROLE_ADMIN is based on ROLE_DISPATCHER

**Comment by vladimir-8** [comment]
Tried adding a new vehicle but got this error:
Screenshot 2024-07-12 at 08 30 39

**Comment by vladimir-8** [comment]
Also, I saw you changed a global Modal's style. So the modals in foodtech now look like this. Please check with Vincent if that's okay

Screenshot 2024-07-12 at 08 57 18

File: src/Entity/Trailer.php (hunk 1 above)
Comment (Atala) [reply src/Entity/Trailer.php]: cleaned

**Comment by vladimir-8** [comment]
Should Warehouses section have a different icon?

Screenshot 2024-07-18 at 11 19 37

**Comment by vladimir-8** [comment]
Not the biggest issue at the moment, but if you have time we can make it a bit tidier:

Screenshot 2024-07-18 at 11 21 11

**Review by vladimir-8 — APPROVED** [review]
All works now. I added links to vehicle and trailer issues, could you please add a link to warehouse issue and the designs?

**Comment by vladimir-8** [comment]
Also, don't forget to check tests 😄 Some a still failing

**Pull request merged**